lib/rasn1/model.rb

```diff
@@ -588,10 +588,19 @@ def sequence_to_h(seq)
         name = case el
                when Model
                  @elements.key(el)
+               when Wrapper
+                 @elements.key(el.element)
                else
                  el.name
                end
-        [name, private_to_h(el)]
+
+        if name.nil?
+          # $stderr.puts "name is nil"
+          # name = @elements[:ticket].instance_variable_get(:@elements).key(el.element)
```

Comment:
Adding this line back in allows the tests to pass; Might require additional changes to https://github.com/sdaubert/rasn1/pull/22